### DISCUSSION TITLE
Underline offset

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#2373] [Q&A] [answered]
I want to increase space between symbols and underline.

In gtk.css (~/.config/gtk-3.0/) I write

```
#zim-pageview text {
 text-underline-offset: 3px;
 }
```
but it does not work.

How should I do it right ?

**Comment by jaap-karssenberg** [comment, marked as answer]
Gtk only support a sub-set of the full CSS spec. Please see documentation
here: Gtk – 3.0: CSS Properties
<https://docs.gtk.org/gtk3/css-properties.html>

--- Jaap
